src/winfile.h

```diff
@@ -1183,6 +1185,9 @@ Extern TCHAR        szUILanguage[]          EQ( TEXT("UILanguage") );
 Extern TCHAR        szEditorPath[]          EQ( TEXT("EditorPath"));
 Extern TCHAR        szMirrorContent[]       EQ( TEXT("MirrorContent") );
 Extern TCHAR        szCachedPath[]          EQ( TEXT("CachedPath"));
+Extern TCHAR        szCachedRootLower[MAXPATHLEN];

```

Comment:
Should be passed as a parameter to BuildDirectoryBagOValues instead of a global.